src/controllers/accommodationController.js

```diff
@@ -17,8 +18,16 @@ const { serverError } = messages;
  * @returns {Object} - custom response
 */
 const createAccommodation = async (req, res) => {
+  const { TRAVEL_ADMIN, SUPER_ADMIN, ACCOMMODATION_SUPPLIER } = roles;
+  const checkRole = TRAVEL_ADMIN || SUPER_ADMIN || ACCOMMODATION_SUPPLIER;
```

Comment:
LET TRY USING AUTHORIZE MIDDLEWARE ONLY AT FIRST AND SEE IF IT STILL WORKS

Reply:
It works